core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -7615,6 +7636,15 @@ public static final class Name {
     public static final String TABLE_LOAD_DEFAULT_REPLICATION =
         "alluxio.table.load.default.replication";
 
+    public static final String HADOOP_SECURITY_AUTHENTICATION =
+            "alluxio.hadoop.security.authentication";
+
+    public static final String HADOOP_KRB5_CONF_KEY_FILE =
+            "alluxio.java.security.krb5.conf";
```

Comment:
using hadoop instead of java seems to be more consistent
```suggestion
            "alluxio.hadoop.security.krb5.conf";
```